packages/infrastructure/cluster-issuer.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
What does this file do? Who reference it?

Reply:
Nothing yet. Should not be her. I will remove